src/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,38 @@
+import { ChangeEvent } from 'react';
+
+import style from './FormInput.module.scss';
+
+type FormInputProps = {
+  type: string;
```

Comment:
We could improve the type of the type. It is surely not a string, also it does not have to be required, the default is `text`

```suggestion
type InputProps = ComponentProps<'input'>

//.....

  type: InputProps['type']
```

We can do the same for onChange